src/main/resources/templates/elasticsearch/caseMatching/dormant_date_range_query.json

```diff
@@ -6,24 +6,36 @@
       "must_not": { "match": { "state": "Dormant" }},
       "filter": [
         {
-          "range": {
-            "last_modified": {
-              "gte": ":fromDate",
-              "lte": ":toDate"
-            }
+          "bool": {
+            "should": [
+              {
+                "range": {
+                  "last_modified": {
+                    "gte": ":fromDate",
+                    "lte": ":toDate"
+                  }
+                }
+              },
+              {
+                "range": {
+                  "data.lastModifiedDateForDormant": {
+                    "gte": ":fromDate",
+                    "lte": ":toDate"
+                  }
+                }
+              }
+            ],
+            "minimum_should_match": 1
           }
         },
         {
           "bool": {
             "should": [
-              { "match": { "state": "BOCaseMatchingExamining" }},
               { "match": { "state": "BOCaseMatchingIssueGrant" }},
               { "match": { "state": "BOCaseQA" }},
               { "match": { "state": "BOReadyToIssue" }},
               { "match": { "state": "BORegistrarEscalation" }},
               { "match": { "state": "BOCaseStopped" }},
-              { "match": { "state": "BOExamining" }},
-              { "match": { "state": "BOReadyForExamination" }},
               { "match": { "state": "CasePrinted" }},
               { "match": { "state": "CasePaymentFailed" }},
```

Comment:
Can we remove pre-submission states, CasePaymentFailed and CaseCreated states as well?